scripts/ErrorReporter/errorreport.py

```diff
@@ -15,21 +15,39 @@
 else:
     raise RuntimeError("Unknown QT version: {}".format(qtpy.QT_VERSION))
 
-from qtpy import QtCore, QtGui, QtWidgets # noqa: E402
-from qtpy.QtCore import Signal # noqa: E402
-from qtpy.QtWidgets import QMessageBox # noqa: E402
-from mantidqt.utils.qt import load_ui # noqa: E402
+from qtpy import QtCore, QtGui, QtWidgets  # noqa: E402
+from qtpy.QtCore import Signal, QUrl  # noqa: E402
+from qtpy.QtGui import QDesktopServices  # noqa: E402
+from qtpy.QtWidgets import QMessageBox, QTextEdit  # noqa: E402
+from mantidqt.utils.qt import load_ui, import_qt  # noqa: E402
+
+MantidDesktopServices = import_qt('.._common', package='mantidqt.widgets', attr='MantidDesktopServices')
 
 ErrorReportUIBase, ErrorReportUI = load_ui(__file__, 'errorreport.ui')
 
 
 class CrashReportPage(ErrorReportUIBase, ErrorReportUI):
     action = Signal(bool, int, str, str, str)
     quit_signal = Signal()
+    default_plain_text = ("Please enter any additional information about your problems. "
+                          "(Max 3200 characters)\n"
+                          "\n"
+                          "For example:\n"
+                          "Error messages on the screen\n "
+                          "A script that causes the problem\n "
```

Comment:
There are some spaces after the new line characters here that are indenting the lines slightly